src/main.lua

```diff
@@ -2610,4 +2610,21 @@ addEventingHandler("allPaginatedGatewayVaults", utils.hasMatchingTag("Action", "
 	Send(msg, { Target = msg.From, Action = "All-Gateway-Vaults-Notice", Data = json.encode(result) })
 end)
 
+-- Temporary handler for posting epoch distribution notices (make sure to remove this once previous epochs are distributed)
```

Comment:
May want to add TODO here for easier discovery later

Reply:
[c229f56](https://github.com/ar-io/ar-io-network-process/pull/328/commits/c229f56199a82931cd0951f2efbc643a6add8525)